Makefile

```
@@ -37,12 +41,17 @@ test: test-unit
+# Tests REPL mechanics against a real model. LLM-dependent tests SKIP gracefully.
+live: all
+	@bash e2e/test_live.sh $(BUILD_DIR)/llama-cli
+
 test-e2e: build
 	@echo "==> E2E tests"
-	@for t in e2e/*.sh; do echo "Running $$t"; bash "$$t" $(BUILD_DIR)/llama-cli || exit 1; done
+	@for t in e2e/*.sh; do case "$$t" in *test_live*|*helpers*) continue;; esac; echo "Running $$t"; bash "$$t" $(BUILD_DIR)/llama-cli || exit 1; done
 	@echo "E2E tests passed."
 
-check: all test e2e format-check
+check: all test e2e format index
```

checkmake (0.2.2)

[warning] Target body for "check" exceeds allowed length of 5 (26).

(maxbodylength)